CMakeLists.txt

```
@@ -3111,6 +3112,15 @@ if (SDL_ASAN)
   endif()
 endif()
 
+if (SDL_CLANG_TIDY)
+    find_program(CLANG_TIDY clang-tidy REQUIRED)
```

For SDL3, we're probably going to bump the minimum required verson to 3.16.
```suggestion
    cmake_minimum_required(VERSION 3.6)
    find_program(CLANG_TIDY clang-tidy REQUIRED)
```

---

I also removed the `REQUIRED` on `find_program` as it's (surprisingly) a 3.18 feature, opting to check the output variable instead.